.github/workflows/installdaprwin.yml

```diff
@@ -0,0 +1,62 @@
+# Runs the Dapr Windows Powershell Install Script
+
+name: Run Dapr Windows Install Script
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'install/**'
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'install/**'
+  release:
+    types:
+      - created
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  install:
+    runs-on: windows-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Install Dapr
+      - name: Install DAPR CLI
+        run: powershell "install/install.ps1"
+      
+      - name: Verify installed CLI version
+        uses: Amadevus/pwsh-script@v2.0.1
+        with:
+          script: |
+            $Env:Path += ";c:\dapr"
```

Comment:
@berndverst Isn't the script already setting the env path?

Reply:
It doesn't work without this line because in Github Actions the Path isn't updated between steps unless you explicitly tell it to do so in my experience.

All I can say is that this was required in my testing and it certainly doesn't hurt. So why not approve unless there is something else?